src/libraries/System.Net.HttpListener/src/System/Net/Windows/CookieExtensions.cs

```diff
@@ -13,7 +13,7 @@ internal static class CookieExtensions
     {
         private static Func<Cookie, string> s_toServerStringFunc;
 
-        [PreserveDependency("ToServerString", "System.Net.Cookie", "System.Net.Primitives")]
+        [DynamicDependency("ToServerString", "System.Net.Cookie", "System.Net.Primitives")]
```

Comment:
Can this use typeof(Cookie)? Same below.

Reply:
The attribute is should not be needed at all